mmyolo/models/task_modules/assigners/batch_atss_assigner.py

```diff
@@ -1,254 +1,368 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Tuple
```

Comment:
yolov6 is an anchor-free algorithm, so it is best not to have the anchor field and should use prior uniformly